R/orderly_run_session.R

```diff
@@ -0,0 +1,26 @@
+orderly_run_session <- function(name = NULL, parameters = NULL, envir = NULL,
```

Comment:
How do you run this? Does it need exporting?

Reply:
Oops, of course!